src/Movies/Movies.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import Card from '../Card/Card';
 
 const Movies = (props) => {
-  console.log(props)
+  // console.log(props)
```

Comment:
Remember to use your React Dev Tools - they will show you the up to date props and state of any component :)